src/modules/handleTodo.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable linebreak-style */
```

Comment:
Could you avoid disabling linter checks in line? They represent coding standards that you share with your team and most of the time enforce the best coding standards.

There are three other occurrences.